tools/initToast/initToast.ps1

```diff
@@ -0,0 +1,32 @@
+param([string]$name, [string]$icon, [Int32]$port=80)
```

Comment:
tf do we need this file for?

Reply:
I think it's used for some sort of notifications of some kind, i have not dived into it